web/assets/javascripts/application.js

```diff
@@ -50,27 +66,44 @@ function updateFuzzyTimes(locale) {
   t.cancel();
 }
 
-function updatePage(url) {
+function updateLivePollButton() {
+  if (localStorage.sidekiqLivePoll == "enabled") {
+    $('.live-poll-stop').show();
+    $('.live-poll-start').hide();
+  } else {
+    $('.live-poll-start').show();
+    $('.live-poll-stop').hide();
+  }
+}
+
+function livePollCallback() {
+  clearTimeout(livePollTimer);
+
   $.ajax({
-    url: url,
+    url: document.url,
     dataType: 'html'
-  }).done(function(data) {
-    $data = $(data)
+  }).done(
+    replacePage
+  ).complete(
+    scheduleLivePoll
+  )
+}
 
-    var $page = $data.filter('#page')
-    $('#page').replaceWith($page)
+function scheduleLivePoll() {
+  let ti = parseInt(localStorage.sidekiqTimeInterval) || 5000;
```

Comment:
Changed to `5000` to match the default value for polling on the dashboard page (both use `localStorage.sidekiqTimeInterval` when it's set).